docs/md/dev-wgs.md

```diff
@@ -50,3 +52,67 @@ The `unmapped_bam_suffix`
 is almost always `.unmapped.bam`,
 so that is its default value
 unless it is specified.
+
+The `reference_fasta_prefix` can be used to override
+the [default value](https://github.com/broadinstitute/wfl/blob/master/api/src/wfl/references.clj#L7) used by this module: 
+"gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38"
+
+## Usage
+
+###Create Workload: `/api/v1/create`
+Creates a WFL workload. Before processing, confirm that the WFL and Cromwell service accounts have
+at least read access to the input files.
+```
+curl --location --request POST 'https://dev-wfl.gotc-dev.broadinstitute.org/api/v1/create' \
+--header 'Authorization: Bearer '$(gcloud auth print-access-token) \
+--header 'Content-Type: application/json' \
+--data-raw '{
+  "cromwell": "https://cromwell-gotc-auth.gotc-dev.broadinstitute.org",
```

Comment:
Is there a default cromwell for each WFL environment?

Reply:
Yes, so this one is the Cromwell for the `gotc-dev` environment